core/domain/stats_jobs_one_off.py

```diff
@@ -1238,9 +1239,9 @@ def reduce(key, stringified_values):
                         key, state_name, all_state_hit[state_name]),)
 
 
-class MissingStatsModelRegenerationOneOffJob(
+class MissingV2StatsModelRegenerationOneOffJob(
```

Comment:
Why not keep them consistent: `RegenerateMissingV2StatsModelsOneOffJob`?

Reply:
I agree with @brianrodri -- let's keep the names consistent (assuming the jobs are doing the same thing except that one operates on v1 models and the other operates on v2 models?)

Reply:
Ah sure! The current naming was a hacky way to distinguish the jobs before. I'll rename them!

Reply:
Done!